utils/guidelines_xslt/odd2html/config.xsl

```diff
@@ -136,8 +136,8 @@
             <xd:p>The dist folder, in which the final results of the XSLT are stored.</xd:p>
         </xd:desc>
     </xd:doc>
-    <xsl:variable name="dist.folder" select="$cleaned.basedir || 'dist/guidelines/web/'" as="xs:string"/>
-
+    <xsl:variable name="dist.folder" select="$cleaned.basedir || 'dist/guidelines/' || $revision || '/web/'" as="xs:string"/>
```

Comment:
this is the interesting part, as it would allow to build for different revisions. But maybe it would be better to build into `'dist/' || $revision`?